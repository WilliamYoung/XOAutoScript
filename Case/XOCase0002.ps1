start-transcript -path ./Results/XOCase0002_Results.txt
rename-group grouptest1 grouptest2 
stop-transcript