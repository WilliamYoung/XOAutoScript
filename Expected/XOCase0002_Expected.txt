Group grouptest1 renamed!